package.json

```diff
@@ -17,11 +17,14 @@
   "files": [
     "src"
   ],
-  "repository": "eslint/eslint-plugin-markdown",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": "eslint/markdown",
```

Comment:
Just to check, since it wasn't mentioned in https://github.com/eslint/eslint-plugin-markdown/issues/260 - are we going to rename this repo?

Reply:
Yes.